### ISSUE TITLE
Fix Alt Text Label on Card on Storybook

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/ncids#1863]
### Issue description
The alt text label on the image for the default feature card, Card in Text Left, and Card in Text Right on the NCIDS Storybook uses `atl="CRCHD" diversity training` instead of `atl="CRCHD" diversity training`. This needs to be fixed because it is failing accessibility testing due to the typo.

### Steps to reproduce the issue

1.  Go to [NCIDS Storybook on the default Card story](https://designsystem-dev.cancer.gov/develop/storybook/?path=/story/components-card-default--default), the [NCIDS Storybook of the Card in Text Left](https://designsystem-dev.cancer.gov/pr-1857/storybook/?path=/story/components-card-test-cases--card-in-text-left), or the [NCIDS Storybook of the Card in Text Right](https://designsystem-dev.cancer.gov/pr-1857/storybook/?path=/story/components-card-test-cases--card-in-text-right)
2. Inspect the image element of the card
3. See that the code is `<img src="static/media/news1-16x9.66f0093a.jpg" atl="CRCHD" diversity="" training="">` instead of using `alt`.

### What's the expected result?
- The code for the default card storybook should use `alt` instead of `atl`

### What's the actual result?
- Accessibility is failing due to the typo of `atl` instead of `alt`.

### Additional details / screenshot
![image](https://github.com/user-attachments/assets/033926d5-6359-4b4d-8b3b-4fefcca751b3)
